File: chap06/backend/src/main/java/com/mooc/backend/enumerations/PageStatus.java
package com.mooc.backend.enumerations;

public enum PageStatus {
    DRAFT("草稿"),
    PUBLISHED("已发布"),
    ARCHIVED("已归档");

    private final String value;

    PageStatus(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: chap08/frontend/models/lib/enums/page_status.dart
/// 页面状态
/// - draft: 草稿
/// - published: 已发布
/// - archived: 已归档，过期后会自动归档
enum PageStatus {
  draft('Draft'),
  published('Published'),
  archived('Archived');

  final String value;

  const PageStatus(this.value);
}
